.readthedocs.yml

```
@@ -5,7 +5,7 @@ formats: [pdf]
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3.12"
```

Or for the last stable CPython version:

```suggestion
    python: "3"
```

https://docs.readthedocs.io/en/stable/config-file/v2.html#build-tools-python